SPECS/kata-containers/match-Guest-and-Host-cgroup-setup.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN a/src/runtime/arch/amd64-options.mk b/src/runtime/arch/amd64-options.mk
+--- a/src/runtime/arch/amd64-options.mk	2022-08-16 14:41:20.000000000 -0700
++++ b/src/runtime/arch/amd64-options.mk	2022-08-24 17:28:14.113345432 -0700
+@@ -6,7 +6,7 @@
+ # Intel x86-64 settings
+ 
+ MACHINETYPE := q35
+-KERNELPARAMS :=
++KERNELPARAMS := systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0
```

Comment:
Looking at this a bit closer - while this method works, is it possible to pass the `KERNELPARAMS` option in the spec's `%global runtime_make_vars` ? I'm thinking if passing the `KERNELPARAMS` option at make invocation time does work, then we could eliminate the need for this patch altogether.